locopy/database.py

```diff
@@ -220,7 +224,11 @@ def to_dataframe(self, size=None):
 
         if len(fetched) == 0:
             return None
-        return pandas.DataFrame(fetched, columns=columns)
+
+        if df_type == "pandas":
```

Comment:
Sorry for not catching this earlier - we should have some type of input validation on `df_type` as the first thing in this method so that if it's not `pandas` or `polars`, it raises a ValueError. Same in `locopy.snowflake`.

E.g. if I do 

```python
with Snowflake(**creds, dbapi=snowflake.connector) as cmd:
    cmd.execute("select * from table limit 5;")
    df = cmd.to_dataframe(df_type="fake")
```

it will return `None` silently. We want this to fail loudly right at the start. We should also unit test this case as well.

Reply:
just added